gradle.properties

```diff
@@ -0,0 +1,6 @@
+org.gradle.parallel=true
```

Comment:
This is the default

Reply:
Where did you find this? I kept searching and couldn't find it.